### PR TITLE
README. fixes #21

### LINKED ISSUE #21: Consider improving README.md
I'm personally a fan of [README-driven-development](http://tom.preston-werner.com/2010/08/23/readme-driven-development.html).

### PR DESCRIPTION
**Pull request by princiya**
Issue #21 

Work in Progress :) Everyone is free to edit!

### CODE REVIEW

**Comment by princiya** [comment]
What else do we add? @jonathanKingston @groovecoder @biancadanforth

**Comment by jonathanKingston** [comment]
LGTM... I can't see anything obvious missing however I'm the worst at English and doing this kinda thing. @groovecoder is better. We can do follow ups for any extra :)

File: README.md [hunk 1]
```diff
@@ -1,2 +1,37 @@
 # lightbeam-we
-Web Extension version of the Mozilla Lightbeam add-on
+This is the web extension version of the Mozilla Lightbeam add-on for visualizing HTTP requests between websites in real time.
+
+The Mozilla Lightbeam extension is a key tool for Mozilla to educate the public about privacy
+
+## Quick Start
+
+* Clone the repository
```
Comment (biancadanforth) [inline README.md]: I would say a more typical workflow might be to fork the repo, clone the fork (becomes `origin`) and sync the repo as `upstream` in Git? This is the workflow @jkt advised for me for my development work on Lightbeam anyway! It means less notifications/noise if people are experimenting and making DNM commits, I believe.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,37 @@
...
+    - This will launch Firefox and install the extension automatically.
+    - This tool gives you some additional development features such as [automatic reloading](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/Getting_started_with_web-ext#Automatic_extension_reloading).
+
+## Development Guide
+
+* Run `npm install`.
+* Run `npm run test` to check that everything is OK.
+  * If you have installed `eslint` globally, you will have to install globally the following `eslint` plugins too:
+    - `eslint-plugin-json`
+    - `eslint-plugin-mocha`
```
Comment (biancadanforth) [inline README.md]: Nice detail here!

File: README.md (hunk 1 above)
Comment (princiya) [inline README.md]: ok, writing this in the follow up PR for readme. Thanks

**Pull request merged**